src/main/resources/asset_exchange_schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "asset_exchange.asset": {
+    "transaction": true,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
```

Comment:
Please get familiar with clustering keys (no need to update the application) since it's one of the tricky parts in Scalar DB data model.